From 1920bb1f8022202530eeae3e488d6f5156799faf Mon Sep 17 00:00:00 2001
From: Tian Tao <tiantao6@hisilicon.com>
Date: Thu, 8 Apr 2021 17:05:21 +0900
Subject: [PATCH] cifsd: remove unused including <linux/version.h>
Git-commit: 1920bb1f8022202530eeae3e488d6f5156799faf
Patch-mainline: v5.15-rc1
References: bsn#12
Modified-by-SEL: No


Remove including <linux/version.h> that don't need it.

Signed-off-by: Tian Tao <tiantao6@hisilicon.com>
Signed-off-by: Zhiqi Song <songzhiqi1@huawei.com>
Signed-off-by: Namjae Jeon <namjae.jeon@samsung.com>
Signed-off-by: Steve French <stfrench@microsoft.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 fs/cifsd/crypto_ctx.c        | 1 -
 fs/cifsd/glob.h              | 1 -
 fs/cifsd/mgmt/tree_connect.c | 1 -
 fs/cifsd/mgmt/user_session.c | 1 -
 fs/cifsd/mgmt/user_session.h | 1 -
 fs/cifsd/misc.c              | 1 -
 fs/cifsd/vfs.c               | 1 -
 fs/cifsd/vfs_cache.h         | 1 -
 8 files changed, 8 deletions(-)

diff --git a/fs/cifsd/crypto_ctx.c b/fs/cifsd/crypto_ctx.c
index 2c31e8b32de7..8322b0f7a7fc 100644
--- a/fs/cifsd/crypto_ctx.c
+++ b/fs/cifsd/crypto_ctx.c
@@ -9,7 +9,6 @@
 #include <linux/slab.h>
 #include <linux/wait.h>
 #include <linux/sched.h>
-#include <linux/version.h>
 
 #include "glob.h"
 #include "crypto_ctx.h"
diff --git a/fs/cifsd/glob.h b/fs/cifsd/glob.h
index d0bc6edd0477..9d70093a837a 100644
--- a/fs/cifsd/glob.h
+++ b/fs/cifsd/glob.h
@@ -8,7 +8,6 @@
 #define __KSMBD_GLOB_H
 
 #include <linux/ctype.h>
-#include <linux/version.h>
 
 #include "unicode.h"
 #include "vfs_cache.h"
diff --git a/fs/cifsd/mgmt/tree_connect.c b/fs/cifsd/mgmt/tree_connect.c
index d3f28b10db4b..b9cd8fc46e5e 100644
--- a/fs/cifsd/mgmt/tree_connect.c
+++ b/fs/cifsd/mgmt/tree_connect.c
@@ -5,7 +5,6 @@
 
 #include <linux/list.h>
 #include <linux/slab.h>
-#include <linux/version.h>
 #include <linux/xarray.h>
 
 #include "../buffer_pool.h"
diff --git a/fs/cifsd/mgmt/user_session.c b/fs/cifsd/mgmt/user_session.c
index bd5789b7e08e..52c5c036ecf9 100644
--- a/fs/cifsd/mgmt/user_session.c
+++ b/fs/cifsd/mgmt/user_session.c
@@ -6,7 +6,6 @@
 #include <linux/list.h>
 #include <linux/slab.h>
 #include <linux/rwsem.h>
-#include <linux/version.h>
 #include <linux/xarray.h>
 
 #include "ksmbd_ida.h"
diff --git a/fs/cifsd/mgmt/user_session.h b/fs/cifsd/mgmt/user_session.h
index 1a97c851f2fc..ad5c0430b62a 100644
--- a/fs/cifsd/mgmt/user_session.h
+++ b/fs/cifsd/mgmt/user_session.h
@@ -7,7 +7,6 @@
 #define __USER_SESSION_MANAGEMENT_H__
 
 #include <linux/hashtable.h>
-#include <linux/version.h>
 #include <linux/xarray.h>
 
 #include "../smb_common.h"
diff --git a/fs/cifsd/misc.c b/fs/cifsd/misc.c
index b6f3f0818217..cbaaecf2eca1 100644
--- a/fs/cifsd/misc.c
+++ b/fs/cifsd/misc.c
@@ -5,7 +5,6 @@
  */
 
 #include <linux/kernel.h>
-#include <linux/version.h>
 #include <linux/xattr.h>
 #include <linux/fs.h>
 
diff --git a/fs/cifsd/vfs.c b/fs/cifsd/vfs.c
index d3882208a259..5985d2d1f276 100644
--- a/fs/cifsd/vfs.c
+++ b/fs/cifsd/vfs.c
@@ -9,7 +9,6 @@
 #include <linux/uaccess.h>
 #include <linux/backing-dev.h>
 #include <linux/writeback.h>
-#include <linux/version.h>
 #include <linux/xattr.h>
 #include <linux/falloc.h>
 #include <linux/genhd.h>
diff --git a/fs/cifsd/vfs_cache.h b/fs/cifsd/vfs_cache.h
index 318dcb1a297a..8226fdf882e4 100644
--- a/fs/cifsd/vfs_cache.h
+++ b/fs/cifsd/vfs_cache.h
@@ -6,7 +6,6 @@
 #ifndef __VFS_CACHE_H__
 #define __VFS_CACHE_H__
 
-#include <linux/version.h>
 #include <linux/file.h>
 #include <linux/fs.h>
 #include <linux/rwsem.h>
-- 
2.33.0

